### PR TITLE
Add PR template for BPG

### LINKED ISSUE #77: PR Request template
The current PR request template uses the pipeline PR template. BPG should have its own PR template with example code section rather than pipeline test runs.

### PR DESCRIPTION
**Pull request by stefaneng**
# Description
Adds BPG specific PR template

### Closes #77

# Checklist

- [x] This PR does NOT contain [PHI](https://ohrpp.research.ucla.edu/hipaa/) or germline genetic data.  A repo may need to be deleted if such data is uploaded. Disclosing PHI is a [major problem](https://healthitsecurity.com/news/ucla-health-reaches-7.5m-settlement-over-2015-breach-of-4.5m).
- [x] This PR does NOT contain molecular files, compressed files, output files such as images (*e.g.* `.png`, .`jpeg`), `.pdf`, `.RData`, `.xlsx`, `.doc`, `.ppt`, or other non-plain-text files.  To automatically exclude such files using a [.gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files) file, see [here](https://github.com/uclahs-cds/template-base/blob/main/.gitignore) for example.
- [x] I have read the [code review guidelines](https://confluence.mednet.ucla.edu/display/BOUTROSLAB/Code+Review+Guidelines) and the [code review best practice on GitHub check-list](https://confluence.mednet.ucla.edu/display/BOUTROSLAB/Code+Review+Best+Practice+on+GitHub+-+Check+List).
- [x] I have set up or verified the `main` branch protection rule following the [github standards](https://confluence.mednet.ucla.edu/pages/viewpage.action?spaceKey=BOUTROSLAB&title=GitHub+Standards#GitHubStandards-Branchprotectionrule) before opening this pull request.
- [x] The name of the branch is meaningful and well formatted following the [standards](https://confluence.mednet.ucla.edu/display/BOUTROSLAB/Code+Review+Best+Practice+on+GitHub+-+Check+List), using [AD_username (or 5 letters of AD if AD is too long)]-[brief_description_of_branch].
- [x] I have added the major changes included in this pull request to the `CHANGELOG.md` under the next release version or unreleased, and updated the date.

### CODE REVIEW

**Comment by dan-knight** [comment]
Does reprex include screenshots of the results? It might be good to phrase the requirements for the test results section more generally, then suggest reprex as a helpful tool. Something like

> Include a small working example and a screenshot of the results if applicable. You can use the `reprex` package to automatically generate example code,"

**Comment by stefaneng** [comment]
> Does reprex include screenshots of the results? It might be good to phrase the requirements for the test results section more generally, then suggest reprex as a helpful tool. Something like
> 
> > Include a small working example and a screenshot of the results if applicable. You can use the `reprex` package to automatically generate example code,"

Good idea.

Reprex includes a screenshot as well so I think your instructions sound good

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+-   [ ] I have read the [code review guidelines](https://confluence.mednet.ucla.edu/display/BOUTROSLAB/Code+Review+Guidelines) and the [code review best practice on GitHub check-list](https://confluence.mednet.ucla.edu/display/BOUTROSLAB/Code+Review+Best+Practice+on+GitHub+-+Check+List).
+
+-   [ ] I have set up or verified the `main` branch protection rule following the [github standards](https://confluence.mednet.ucla.edu/pages/viewpage.action?spaceKey=BOUTROSLAB&title=GitHub+Standards#GitHubStandards-Branchprotectionrule) before opening this pull request.
+
+-   [ ] The name of the branch is meaningful and well formatted following the [standards](https://confluence.mednet.ucla.edu/display/BOUTROSLAB/Code+Review+Best+Practice+on+GitHub+-+Check+List), using [AD_username (or 5 letters of AD if AD is too long)]-[brief_description_of_branch].
+
+-   [ ] I have added the major changes included in this pull request to the `NEWS` under the next release version or unreleased, and updated the date.
```
Comment (dan-knight) [inline .github/pull_request_template.md:21]: One last thing. We should probably also mention `DESCRIPTION`. I'm thinking something like "I have also updated the version number and date in `DESCRIPTION`.

Comment (stefaneng) [reply .github/pull_request_template.md:21]: I'm actually a bit unsure of the workflow here. Do we update the version every PR or do we have multiple bug fixes in `7.0.X`?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -20,6 +20,8 @@
 
 -   [ ] I have added the major changes included in this pull request to the `NEWS` under the next release version or unreleased, and updated the date.
 
+-   [ ] I have updated the version number in `DESCRIPTION` according to [semantic versioning](https://semver.org/) rules.
```
Comment (dan-knight) [inline .github/pull_request_template.md:23]: Hate to be pedantic, but I wonder if we should combine these for clarity. I see it all as one step, as the date and version number should be consistent between `NEWS` and `DESCRIPTION`. I'm hoping that we can teach new contributors through this checklist. If we get this right, we'd avoid a lot of PR review comments.

Comment (stefaneng) [reply .github/pull_request_template.md]: I merged the NEWS and DESCRIPTION lines

**Review by aholmes — APPROVED** [review]

**Pull request merged**